packages/next-mdx/package.json

```diff
@@ -10,5 +10,8 @@
   "peerDependencies": {
     "@mdx-js/loader": ">=0.15.0",
     "@mdx-js/react": "*"
+  },
+  "dependencies": {
+    "source-map": "^0.7.0"
```

Comment:
Can we make this a `devDependency` and `ncc` the relevant file?

Reply:
`@mdx-js/loader` brings this anyway (https://github.com/vercel/next.js/pull/41919/files#diff-7b9eb4f98eac741f3a0f77cd2a82a19db9f6adaaf8805ca740d6bba439598cedR11), so I thought it'd better to leave it and share across while this is in experimental state. If we bring this as ncc-compiled, there'll be techinically 2 set of source-map between compiled vs mdx-js's transitive deps.

Reply:
Ah I see yeah if it's already present as a runtime dep we can leave for now